packages/pd-console/src/server/models/SampleConsoleModel.ts

```
@@ -0,0 +1,216 @@
+    if (input.decision === 'approved') {
+      const transitioned = await mgr.transitionCandidateStatus(sampleId, 'pending', 'consumed');
+      if (!transitioned) {
+        throw new Error('Failed to transition sample status to consumed');
+      }
+      try {
+        const intake = await this.getIntakeService();
+        await intake.intake(sampleId);
+      } catch (intakeErr) {
+        try {
+          await mgr.updateCandidateStatus(sampleId, { status: 'pending' });
+        } catch (rollbackErr) {
```

ESLint

[error] 'rollbackErr' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] There is no `cause` attached to the symptom error being thrown.

(preserve-caught-error)